src/components/FilterSidebar/components/TemporalFilter/index.tsx

```diff
@@ -25,25 +29,33 @@ const TemporalFilter = ({
         <div className="date-picker-container">
           {/* Date picker for start date */}
           <label htmlFor="startDate">Start</label>
-          <input
+          <DatePicker
+            showMonthDropdown
+            showYearDropdown
             className="date-picker start-date"
+            dateFormat="yyyy/MM/dd"
             id="startDate"
             name="startDate"
-            type="date"
-            value={value.start}
-            onChange={(e) => onStartDateChange(formatDate(e.target.value))}
+            selected={value.start ? new Date(value.start) : new Date()}
+            onChange={(date: Date) => {
+              onStartDateChange(formatDate(date));
+            }}
```

Comment:
I'm not demanding this be done, but single line functions like this can be simplified to:

```
onChange={(date: Date) => onStartDatechange(formatDate(date))}
```

I think this is neater myself, but accept that another commit/ticket later could add more lines and therefore make that redundant. There is an eslint rule to do this with functions, I've just not made it work with inline functions like this. Worth us chatting about it, so we can be consistent in our approaches.